src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -134,14 +157,16 @@ const DataViewTableContent: React.FC<DataViewTableContentProps> = ({
       showToast({ text: strings.DATA_VIEW_COLUMN_NOT_SORTABLE.replace('%1', columnTitle) })
       return
     }
-    // Cycle through ascending/descending/not-sorted (default: id)
-    if (ascending === 'false') {
-      updateQuery({ sortBy: null })
-    } else if (!apiQueries.orderBy) updateQuery({ sortBy: sortColumn })
-    else
+
+    if (ascending === 'false' || !apiQueries.orderBy) {
+      updateQuery({ sortBy: sortColumn })
+      setSortColumn({ title: columnTitle, ascending: true })
+    } else {
```

Comment:
I bit confused with the if statement `if (ascending === 'false'` and the setSortColumn using `ascending: true`

Reply:
"false" refers to the *current* value. So if it's false, change it to true (update query and sortColumn).